### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

**Review by velzevur — APPROVED** [review]

File: test/contracts/channel_htlc.aes [hunk 1]
```diff
@@ -96,7 +96,7 @@ contract ChannelHTLC =
 
     let id : hash = generate_id(sender', state.client, amount', hash_lock')
 
-    require(!recieve_exists(id), "Recieve entry already exists")
+    require(!recieve_exists(id), "Receive entry already exists")
```
Comment (uwiger) [inline test/contracts/channel_htlc.aes:99]: The function name (`recieve_exists()`) is also misspelled.

**Review by uwiger — DISMISSED** [review]

File: test/contracts/channel_htlc.aes [hunk 2]
```diff
@@ -96,7 +96,7 @@ contract ChannelHTLC =
 
     let id : hash = generate_id(sender', state.client, amount', hash_lock')
 
-    require(!recieve_exists(id), "Recieve entry already exists")
+    require(!receive_exists(id), "Receive entry already exists")
```
Comment (uwiger) [inline test/contracts/channel_htlc.aes:99]: You also need to fix the function itself:   `function recieve_exists(id : hash) : bool =` (line 215)

**Review by uwiger — APPROVED** [review]
I verified that the HTLC suite works after the contract code changes in this branch.

**Review by velzevur — APPROVED** [review]

**Comment by velzevur** [comment]
Great work @kianmeng, thanks!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**